plugins/pod-dive.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: pod-dive
+spec:
+  version: "v0.1.2"
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/caiobegotti/pod-dive/releases/download/v0.1.2/pod-dive-amd64-linux.tar.gz
+    sha256: "508a96f1b63484e0ff2790c5a3fa04938466ef7b47103e026cbc2a0ec049e0b2"
+    files:
+    - from: "./pod-dive-amd64-linux"
+      to: "."
+    bin: "pod-dive-amd64-linux"
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/caiobegotti/pod-dive/releases/download/v0.1.2/pod-dive-amd64-darwin.tar.gz
+    sha256: "b243737915199a356130c3867241c0c47313d69855e3e419f6c51cf24c34ff04"
+    files:
+    - from: "./pod-dive-amd64-darwin"
+      to: "."
+    bin: "pod-dive-amd64-darwin"
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/caiobegotti/pod-dive/releases/download/v0.1.2/pod-dive-amd64-windows.exe.zip
+    sha256: "e470c3a86611c3545fa7f5e75479d5f873f3ede6356bf6166bf5a6063703c1d0"
+    files:
+    - from: "./pod-dive-amd64-windows.exe"
+      to: "."
+    bin: "pod-dive-amd64-windows.exe"
+  shortDescription: A kubectl Krew plugin to dive into your Kubernetes nodes workloads
+  homepage: https://github.com/caiobegotti/pod-dive
+  description: |
+    A kubectl Krew plugin to dive into your Kubernetes nodes workloads
```

Comment:
What exactly does "dive" mean? To get people to use your plugin you should explain the use-case or unique features of your plugin.

Reply:
Done!